File: pydss/shared/DynamicsRec.py


class DynamicsRec(object):
    """Variables needed for dynamics and user-written models."""
    # time vars
    h = None
    # time step size in sec for dynamics
    t = None
    # sec from top of hour
    tstart = None
    tstop = None
    # 0 = new time step; 1 = same time step as last iteration
    iterationFlag = None
    # PEAKSNAP, DAILYMODE, YEARLYMODE, MONTECARLO, etc. (see DSSGlobals)
    solutionMode = None
    # FIXME Generate getters and setters

File: core/jdss/src/main/java/com/epri/dss/shared/impl/DynamicsRec.java
package com.epri.dss.shared.impl;

/**
 * Variables needed for dynamics and user-written models.
 */
public class DynamicsRec {

	/** time vars */
	public double h;  // time step size in sec for dynamics
	public double t;  // sec from top of hour
	public double tstart;
	public double tstop;

	/** 0 = new time step; 1 = same time step as last iteration */
	public int iterationFlag;

	/** PEAKSNAP, DAILYMODE, YEARLYMODE, MONTECARLO, etc. (see DSSGlobals) */
	public int solutionMode;

}
